requirements-dev.in

```diff
@@ -1,4 +1,3 @@
-ansible
```

Comment:
This should be replaced by `ansible-core`, so we can run `ansible-galaxy` in the `Jenkinsfile`.

Reply:
Well we still get ansible-core via ansible-lint, but I agree that it would be nice to specify it explicitly.